openlane/scripts/openroad/sta/corner.tcl

```diff
@@ -65,7 +65,7 @@ puts "\n========================================================================
 puts "report_checks -path_delay max (Setup)"
 puts "============================================================================"
 puts "======================= [$corner name] Corner ===================================\n"
-report_checks -sort_by_slack -path_delay max -fields {slew cap input nets fanout} -format full_clock_expanded -group_count 1000 -corner [$corner name]
+report_checks -sort_by_slack -path_delay max -fields {slew cap input nets fanout} -format full_clock_expanded -group_count 1000 -endpoint_count 1000 -corner [$corner name]
```

Comment:
Why just this one?

Reply:
Oh crap, that shouldn't have been in there sorry !
This was just a local temporary hack that slipped in.